<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagepreprocessDlg</class>
 <widget class="QDialog" name="ImagepreprocessDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>40</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>520</y>
     <width>641</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>说明</string>
   </property>
   <widget class="QTextBrowser" name="txtInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>621</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>656</x>
     <y>24</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>默认模式</string>
   </property>
   <widget class="QListWidget" name="listWidgetPattern">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>30</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>656</x>
     <y>240</y>
     <width>361</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>加速器模块</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>64</y>
      <width>351</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>29</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>类型</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
